packages/app/src/cli/services/dev/processes/app-logs-polling.ts

```diff
@@ -0,0 +1,74 @@
+import {BaseProcess, DevProcessFunction} from './types.js'
+import {pollAppLogs} from '../../app-logs/poll-app-logs.js'
+import {DeveloperPlatformClient} from '../../../utilities/developer-platform-client.js'
+
+import {createLogsDir} from '@shopify/cli-kit/node/logs'
+
+import {outputDebug} from '@shopify/cli-kit/node/output'
+
+interface SubscribeAndStartPollingOptions {
+  developerPlatformClient: DeveloperPlatformClient
+  appLogsQueryOptions: AppLogsQueryOptions
+}
+
+interface AppLogsQueryOptions {
+  shopIds: [string]
```

Comment:
This would signify that `shopIds` is effectively a tuple with a single string value. I think we want it to be an array of strings, which would be `string[]`.

Reply:
These options are the same as in the gql file so I've removed this and just imported that one, which has `string[]` 👍